www/js/controller/home.controller.js

```diff
@@ -15,4 +18,13 @@ angular.module('vamosJuntas').controller('HomeController', ['$scope', 'placeFact
     placeFactory.addPlace(place);
   };
 
+  $scope.searchAddress = function() {
+    addressFactory.getAutoCompleteFromAddress($scope.search.text).then(function(response) {
```

Comment:
We should add a test for failed request (and handling it).
Do nothing, try again later, let's think in a solution 😊